hardware/power/android.hardware.power@1.3-service.sony.rc

```diff
@@ -0,0 +1,5 @@
+service vendor.power-hal-1-3 /vendor/bin/hw/android.hardware.power@1.3-service.sony
+    class hal
```

Comment:
Add `interface android.hardware.power@1.3::IPower default` here, might be useful in the future, not only for [dynamic hals](https://source.android.com/devices/architecture/hal/dynamic-lifecycle#changes-HAL-definitions)

Reply:
That's a nice one, thanks, I will :D